data/base/base.file_list

```diff
@@ -70,6 +70,7 @@ AUTODEPS:
 bash:
   /bin/{sh,bash}
   /usr/bin/bash
+  s bash usr/bin/sh
```

Comment:
and bin/sh ?

Reply:
usrmerge forgotten?

Reply:
no - hence my comment being:

> on UsrMerged systems we're probably ok, not sure if we have to care for the other case still

Reply:
(might have added that sentence to slow fo you to see it though)

Reply:
This is only for TW, so usrmerge applies always